.travis.yml

```diff
@@ -50,9 +43,9 @@ matrix:
 before_install:
   - if [ $HHVM != 1 ]; then phpenv config-rm xdebug.ini; fi
 
-  - composer self-update
```

Comment:
Checking the build logs i saw travis already runs `composer self-update` at start so we don't need to do it.